src/PhotoNim.nim

```diff
@@ -1,24 +1,28 @@
-import PhotoNim/[geometry, hdrimage, camera]
+import PhotoNim/[geometry, hdrimage, camera, shapes]
 
 import docopt
 from nimPNG import savePNG24
 
 from std/os import splitFile
 from std/streams import Stream, newFileStream, close
-from std/strutils import parseFloat
+from std/strutils import parseFloat, parseInt
 from std/strformat import fmt
-from std/math import pow
-
+from std/math import pow, exp
+from std/times import cpuTime
+import typetraits
 
 let PhotoNimDoc = """PhotoNim: a CPU raytracer written in Nim.
 
 Usage:
     ./PhotoNim pfm2png <input> [<output>] [--alpha=<alpha> --gamma=<gamma>]
-    ./PhotoNim demo cam = <camType>
+    ./PhotoNim demo (perspective|orthogonal) [<output>] [--width=<width> --height=<height> --angle=<angle>]
```

Comment:
Maybe i would set a variable like --alpha to set the camera:
```nim
./PhotoNim demo --camera=<camera> ...
```

Reply:
I don't know here honestly Lollo, i like that you are giving an exclusive choice: you have to type perspective or orthogonal in order to start rendering, i feel like it's safer regarding user errors

Reply:
You can make it 'not optional' by just writing `<camera>` and adding an enum that lists all the possible cameras. The fact is that an user could wrong spell the names of the cameras and the program would then crash.